--- main/src/CaelumPlugin.cpp.old	2011-08-22 14:39:29.000000000 +0200
+++ main/src/CaelumPlugin.cpp	2012-07-08 23:23:01.652985414 +0200
@@ -21,17 +21,17 @@
 #include "CaelumPrecompiled.h"
 #include "CaelumPlugin.h"
 
-template<> Caelum::CaelumPlugin* Ogre::Singleton<Caelum::CaelumPlugin>::ms_Singleton = 0;
+template<> Caelum::CaelumPlugin* Ogre::Singleton<Caelum::CaelumPlugin>::msSingleton = 0;
 
 namespace Caelum
 {
 	CaelumPlugin* CaelumPlugin::getSingletonPtr () {
-        return ms_Singleton;
+        return msSingleton;
     }
 
     CaelumPlugin& CaelumPlugin::getSingleton () {  
-        assert (ms_Singleton);
-        return *ms_Singleton;  
+        assert (msSingleton);
+        return *msSingleton;
     }
 
     extern "C" void CAELUM_EXPORT dllStartPlugin () {
