diff -ur ogreal-old/Demos/Basic_Demo/OgreApp.cpp ogreal-9999/Demos/Basic_Demo/OgreApp.cpp
--- ogreal-old/Demos/Basic_Demo/OgreApp.cpp	2013-09-15 16:42:45.461495344 +0200
+++ ogreal-9999/Demos/Basic_Demo/OgreApp.cpp	2013-09-15 16:43:16.186497430 +0200
@@ -70,7 +70,7 @@
 
 void OgreApp::createScene()
 {
-	sceneMgr->setSkyBox(true, "Sky", 5, 8, 4000);
+	sceneMgr->setSkyBox(true, "Sky");
     sceneMgr->setAmbientLight(ColourValue(0.5f, 0.5f, 0.5f));
     sceneMgr->setShadowTechnique(SHADOWTYPE_STENCIL_ADDITIVE);
 
diff -ur ogreal-old/Demos/Directional_Demo/OgreApp.cpp ogreal-9999/Demos/Directional_Demo/OgreApp.cpp
--- ogreal-old/Demos/Directional_Demo/OgreApp.cpp	2013-09-15 16:42:45.438495342 +0200
+++ ogreal-9999/Demos/Directional_Demo/OgreApp.cpp	2013-09-15 16:43:20.041497692 +0200
@@ -70,7 +70,7 @@
 
 void OgreApp::createScene()
 {
-	sceneMgr->setSkyBox(true, "Sky", 5, 8, 4000);
+	sceneMgr->setSkyBox(true, "Sky");
     sceneMgr->setAmbientLight(ColourValue(0.5f, 0.5f, 0.5f));
 	//sceneMgr->setShadowTechnique(SHADOWTYPE_TEXTURE_ADDITIVE);
 
diff -ur ogreal-old/Demos/Doppler_Demo/OgreApp.cpp ogreal-9999/Demos/Doppler_Demo/OgreApp.cpp
--- ogreal-old/Demos/Doppler_Demo/OgreApp.cpp	2013-09-15 16:42:45.459495344 +0200
+++ ogreal-9999/Demos/Doppler_Demo/OgreApp.cpp	2013-09-15 16:43:23.818497948 +0200
@@ -70,7 +70,7 @@
 
 void OgreApp::createScene()
 {
-	sceneMgr->setSkyBox(true, "Sky", 5, 8, 4000);
+	sceneMgr->setSkyBox(true, "Sky");
     sceneMgr->setAmbientLight(ColourValue(0.5f, 0.5f, 0.5f));
     sceneMgr->setShadowTechnique(SHADOWTYPE_STENCIL_ADDITIVE);
 
diff -ur ogreal-old/Demos/ManySources_Demo/OgreApp.cpp ogreal-9999/Demos/ManySources_Demo/OgreApp.cpp
--- ogreal-old/Demos/ManySources_Demo/OgreApp.cpp	2013-09-15 16:42:45.457495344 +0200
+++ ogreal-9999/Demos/ManySources_Demo/OgreApp.cpp	2013-09-15 16:43:27.930498227 +0200
@@ -70,7 +70,7 @@
 
 void OgreApp::createScene()
 {
-	sceneMgr->setSkyBox(true, "Sky", 5, 8, 4000);
+	sceneMgr->setSkyBox(true, "Sky");
     sceneMgr->setAmbientLight(ColourValue(0.5f, 0.5f, 0.5f));
     sceneMgr->setShadowTechnique(SHADOWTYPE_STENCIL_ADDITIVE);
 
@@ -138,4 +138,4 @@
 	overlay->add2D(panel);
 	panel->addChild(textArea);
 	overlay->show();
-}
\ Kein Zeilenumbruch am Dateiende.
+}
diff -ur ogreal-old/Demos/MultiChannel_Demo/OgreApp.cpp ogreal-9999/Demos/MultiChannel_Demo/OgreApp.cpp
--- ogreal-old/Demos/MultiChannel_Demo/OgreApp.cpp	2013-09-15 16:42:45.460495344 +0200
+++ ogreal-9999/Demos/MultiChannel_Demo/OgreApp.cpp	2013-09-15 16:43:31.642498479 +0200
@@ -70,7 +70,7 @@
 
 void OgreApp::createScene()
 {
-	sceneMgr->setSkyBox(true, "Sky", 5, 8, 4000);
+	sceneMgr->setSkyBox(true, "Sky");
     sceneMgr->setAmbientLight(ColourValue(0.5f, 0.5f, 0.5f));
     sceneMgr->setShadowTechnique(SHADOWTYPE_STENCIL_ADDITIVE);
 
diff -ur ogreal-old/Demos/PlayPen/OgreApp.cpp ogreal-9999/Demos/PlayPen/OgreApp.cpp
--- ogreal-old/Demos/PlayPen/OgreApp.cpp	2013-09-15 16:42:45.458495344 +0200
+++ ogreal-9999/Demos/PlayPen/OgreApp.cpp	2013-09-15 16:43:01.530496435 +0200
@@ -70,7 +70,7 @@
 
 void OgreApp::createScene()
 {
-	sceneMgr->setSkyBox(true, "Sky", 5, 8, 4000);
+	sceneMgr->setSkyBox(true, "Sky");
     sceneMgr->setAmbientLight(ColourValue(0.5f, 0.5f, 0.5f));
     sceneMgr->setShadowTechnique(SHADOWTYPE_STENCIL_ADDITIVE);
 
